contracts/cw20-ics20/src/contract.rs

```diff
@@ -183,6 +249,59 @@ pub fn query_channel(deps: Deps, id: String) -> StdResult<ChannelResponse> {
     })
 }
 
+fn query_config(deps: Deps) -> StdResult<ConfigResponse> {
+    let cfg = CONFIG.load(deps.storage)?;
+    let res = ConfigResponse {
+        default_timeout: cfg.default_timeout,
+        gov_contract: cfg.gov_contract.into(),
+    };
```

Comment:
Why not just returning `Config` itself? Those are the same two only fields.

Reply:
Change from Addr to String.

I kind of got the feel we want to only use String in public APIs and Addr in stuff we have personally verified (in logic or in storage). But maybe this is me just being pedantic

Reply:
Okay, I actually though this is needed internally.